### ISSUE TITLE
Add `quote.created` webhook for publishing Mojaloop payloads

### ISSUE THREAD

**Issue by mkurapov** [interledger/rafiki#1997]
## Context
In order to get a response back from the Mojaloop hub during a Mojaloop payment, whenever we get a mojaloop connector server request on the receiving side (built in #1991), we should publish a new `quote.created` webhook, so that the receiving ASE can properly respond back with any Mojaloop related information (in #1998). 

This ticket involves work for step 4 in the quoting flow:
<img width="1303" alt="Screenshot 2023-10-19 at 15 24 44" src="https://github.com/interledger/rafiki/assets/15069181/5b93eb12-17e9-433c-bbda-45a4da59bd4f">

## Tasks
- [ ] Create a new `quote.created` webhook, that contains the following quote details:
```
{
        id: string
        paymentPointerId: string
        receiver: string
        debitAmount: {
            assetCode
            assetScale
            value
        }
        receiveAmount: {
            assetCode
            assetScale
            value
        }
        expiresAt: string
        estimatedExchangeRate: number
        additionalFields: object
        createdAt: string
        updatedAt: string
}
```
- [ ] Add the new webhook type to the OpenAPI `webhooks.yaml`
- [ ] Update the MASE to handle webhook event (for now, just log it out, and return 200)

Blocked:

~- [ ] Publish the webhook when receiving a Mojaloop connector request (blocked by #1991)~

**Comment by mkurapov** [comment]
Closing in favour of larger Rafiki architecture [overhaul](https://github.com/orgs/interledger/projects/6/views/1?pane=issue&itemId=43212773)
